tests/test_ui_complex_scenarios.py

```
@@ -0,0 +1,841 @@
+
+    def fake_make_on_change():
+        def handler(e):
+            val = e.value
+            if val is not None and val != '':
+                try:
+                    val_str = str(val).strip()
+                    if '.' in val_str:
+                        val = float(val_str)
+                    else:
+                        val = int(val_str)
+                except ValueError: pass
```

Ruff (0.15.18)

[error] Multiple statements on one line (colon)

(E701)